src/components/ColorPicker.tsx

```diff
@@ -5,46 +5,58 @@ import Saturation from "./Saturation";
 
 import styles from "../styles.css";
 import { ColorModel, HSV, ColorPickerBaseProps, AnyColor } from "../types";
-import { equalColorObjects } from "../utils/compare";
 import formatClassName from "../utils/formatClassName";
 
 interface Props<T extends AnyColor> extends Partial<ColorPickerBaseProps<T>> {
   colorModel: ColorModel<T>;
 }
 
+function useEventCallback<T>(event?: (arg: T) => void) {
+  const eventRef = useRef(event);
+
+  useEffect(() => {
+    eventRef.current = event;
+  });
+
+  return useCallback((arg: T) => eventRef.current && eventRef.current(arg), []);
+}
```

Comment:
Nice trick!

Shouldn't we do the same in `HexInput`?
I would move this function to `src/hooks/useEventCallback.js` in order to reuse it in both components.

Reply:
@jeetiss could we reuse the hook inside of HexInput?